lib/workload/stateful/filemanager/filemanager/src/events/aws/collector_builder.rs

```diff
@@ -1,3 +1,6 @@
+use mockall_double::double;
```

Comment:
Why is this not part of collector.rs instead of a separate file?

Reply:
Good point, could be the same file.